osgp/platform/osgp-domain-distributionautomation/src/main/java/org/opensmartgridplatform/domain/da/measurements/MeasurementReport.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.domain.da.measurements;
+
+import java.util.List;
+import java.util.Objects;
+
+public class MeasurementReport {
+
+    private MeasurementReportHeader measurementReportHeader;
+    private List<MeasurementGroup> measurementGroups;
+
+    private MeasurementReport(final Builder builder) {
+        this.measurementReportHeader = builder.measurementReportHeader;
+        this.measurementGroups = builder.measurementGroups;
+    }
+
+    public MeasurementReport(final MeasurementReportHeader header, final List<MeasurementGroup> measurementGroups) {
+        this.measurementReportHeader = header;
+        this.measurementGroups = measurementGroups;
+    }
+
+    public MeasurementReportHeader getHeader() {
+        return this.measurementReportHeader;
+    }
+
+    public List<MeasurementGroup> getMeasurementGroups() {
+        return this.measurementGroups;
+    }
+
+    @Override
+    public boolean equals(final Object obj) {
```

Comment:
Also add an equals and hashCode method to MeasurementReportHeader and MeasurementGroup.

Reply:
Added equals and hashCode methods to MeasurementReportHeader.
MeasurementGroup already had equals and hashCode methods?